index.html

```diff
@@ -2,10 +2,96 @@
 <html>
     <head>
         <title>Tristan Geary</title>
+		
+		<!-- Metadata -->
+		<meta name= "author" content = "Tristan Geary">
```

Comment:
Spaces around equal signs are accepted in html but are not recommended:
https://www.w3schools.com/html/html5_syntax.asp